src/Nervos/CellOutput.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "CellOutput.h"
+
+#include "CommonFunc.h"
+#include "../BinaryCoding.h"
+
+using namespace TW::Nervos;
+
+void CellOutput::encode(Data& data) const {
+    std::vector<Data> dataArray;
```

Comment:
It's seems that this vector only have `3` elements, consider either using `std::array` or using the appropriate constructor from `std::vector` to avoid unnecessary memory allocation (constructor number 4) in order to respect [RAII](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization) pattern

Reply:
Done